web/src/search_suggestion.ts

```diff
@@ -512,45 +523,68 @@ function get_topic_suggestions(last: NarrowTerm, terms: NarrowTerm[]): Suggestio
     // channel:Rome topic:f -> show all Rome topics with a word starting in f
 
     // When narrowed to a channel:
-    //   topic: -> show all topics in current channel
+    //   topic: -> show topics from all subscribed channels with the current channel's
+    //   matching topics listed first.
```

Comment:
I think this comment might be wrong? Or at least it's inconsistent with the `show_topics_from_other_channels` variable name.

Reply:
Ahh, no it's fine, there's two different channels involved.